compiler/qsc_passes/src/capabilitiesck.rs

```diff
@@ -274,6 +277,17 @@ impl<'a> Checker<'a> {
         }
     }
 
+    /// Check that there are no custom measurements in Base Profile.
+    fn check_measurement(&mut self, attrs: &[Attr], span: Span) {
```

Comment:
Even though this is only being used for measurement right now, it's a good spot to include future disallow of attributes based on capabilities (like the `@Reset` we anticipate needing), so I think this can be called `check_attrs` to cover the generic case.

Reply:
Yes, makes sense. Updated.